cost-analyzer/values.yaml

```diff
@@ -533,7 +533,15 @@ kubecostModel:
     #    hosts:
     #      - cost-analyzer-model.local
 
-
+etlUtils:
```

Comment:
Please add some comment here as to what this does and why anyone would use it.

Reply:
good call, added.